### PR TITLE
refactor: modal redesign and fix for transitions

### PR DESCRIPTION
**Pull request by sct**
#### To-Dos

- [ ] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #XXXX

### CODE REVIEW

File: src/components/RequestModal/CollectionRequestModal.tsx [hunk 1]
```diff
@@ -22,8 +21,8 @@ import useSWR from 'swr';
 const messages = defineMessages({
   requestadmin: 'This request will be approved automatically.',
   requestSuccess: '<strong>{title}</strong> requested successfully!',
-  requesttitle: 'Request {title}',
-  request4ktitle: 'Request {title} in 4K',
+  requestcollectiontitle: 'Request Collection',
+  requestcollection4ktitle: 'Request 4K Collection',
```
Comment (TheCatLady) [inline src/components/RequestModal/CollectionRequestModal.tsx:25]:
```suggestion
  requestcollection4ktitle: 'Request Collection in 4K',
```

File: src/components/RequestModal/MovieRequestModal.tsx [hunk 2]
```diff
@@ -21,13 +20,13 @@ const messages = defineMessages({
   requestadmin: 'This request will be approved automatically.',
   requestSuccess: '<strong>{title}</strong> requested successfully!',
   requestCancel: 'Request for <strong>{title}</strong> canceled.',
-  requesttitle: 'Request {title}',
-  request4ktitle: 'Request {title} in 4K',
+  requestmovietitle: 'Request Movie',
+  requestmovie4ktitle: 'Request 4K Movie',
```
Comment (TheCatLady) [inline src/components/RequestModal/MovieRequestModal.tsx:24]:
```suggestion
  requestmovie4ktitle: 'Request Movie in 4K',
```

File: src/components/RequestModal/MovieRequestModal.tsx [hunk 3]
```diff
@@ -21,13 +20,13 @@ const messages = defineMessages({
...
   edit: 'Edit Request',
   approve: 'Approve Request',
   cancel: 'Cancel Request',
-  pendingrequest: 'Pending Request for {title}',
-  pending4krequest: 'Pending 4K Request for {title}',
+  pendingrequest: 'Pending Request',
+  pending4krequest: 'Pending 4K Request',
```
Comment (TheCatLady) [inline src/components/RequestModal/MovieRequestModal.tsx:29]:
```suggestion
  pendingrequest: 'Pending Movie Request',
  pending4krequest: 'Pending 4K Movie Request',
```
The strings for the TV request modal should also be updated.

**Review by TheCatLady — COMMENTED** [review]
Should the delete user modal title also be updated to "Delete User" with the username displayed below?

**Comment by sct** [comment]
> Should the delete user modal title also be updated to "Delete User" with the username displayed below?

I had done this but wiped my changes by mistake and forgot to put them back in. Heh. Adding back!

File: src/components/Common/Button/index.tsx [hunk 4]
```diff
@@ -61,7 +61,7 @@ function Button<P extends ElementTypes = 'button'>(
       break;
     case 'warning':
       buttonStyle.push(
-        'text-white bg-yellow-500 border-yellow-500 hover:bg-yellow-400 hover:border-yellow-400 focus:border-yellow-700 focus:ring-yellow active:bg-yellow-700 active:border-yellow-700'
+        'text-white border border-yellow-500 backdrop-blur bg-yellow-400 bg-opacity-20 hover:bg-yellow-600 hover:border-yellow-400 focus:border-yellow-700 focus:ring-yellow active:bg-yellow-600 active:border-yellow-700'
```
Comment (TheCatLady) [inline src/components/Common/Button/index.tsx:64]: Do we want to update all the buttons? 🤔 Or have solid and translucent options?

Comment (sct) [reply src/components/Common/Button/index.tsx:64]: For now, I would like to keep our primary and delete as is. But we can tweak this and make the style more uniform as we go.

File: src/components/Settings/SettingsJobsCache/index.tsx [hunk 5]
```diff
@@ -324,13 +323,13 @@ const SettingsJobs = () => {
                         setJobEditModal({ isOpen: true, job: job })
                       }
                     >
-                      <PencilIcon />
-                      {intl.formatMessage(globalMessages.edit)}
+                      <PencilIcon className="mr-1 h-5 w-5" />
```
Comment (TheCatLady) [inline src/components/Settings/SettingsJobsCache/index.tsx:326]:
```suggestion
                      <PencilIcon />
```

File: src/components/Settings/SettingsJobsCache/index.tsx [hunk 6]
```diff
@@ -324,13 +323,13 @@ const SettingsJobs = () => {
...
+                      <span>{intl.formatMessage(globalMessages.edit)}</span>
                     </Button>
                   )}
                   {job.running ? (
                     <Button buttonType="danger" onClick={() => cancelJob(job)}>
-                      <StopIcon />
+                      <StopIcon className="mr-1 h-5 w-5" />
```
Comment (TheCatLady) [inline src/components/Settings/SettingsJobsCache/index.tsx:332]:
```suggestion
                      <StopIcon />
```

**Review by TheCatLady — COMMENTED** [review]
SHould also remove from the `Run` button.

**Review by TheCatLady — APPROVED** [review]

**Pull request merged**